release-notes-5.0.0.md

```diff
@@ -1,11 +1,9 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### Breaking Changes
 - SCMOD-12730: Updated to the latest Worker Framework version, which includes the following breaking changes to support pause task functionality.
-  - The `statusCheckUrl` workers use to check the status of a task now points to the `status` endpoint instead of the `isActive` endpoint.
+  - The `statusCheckUrl` workers are used to check the status of a task now points to the `status` endpoint instead of the `isActive` endpoint.
```

Comment:
Perhaps it would be clearer to change it to "The `statusCheckUrl` that workers use to check..."